src/earn/EarnCard.test.tsx

```diff
@@ -132,7 +132,7 @@ describe('EarnCardDiscover', () => {
     expect(queryByTestId('EarnCta')).toBeFalsy()
     expect(getFeatureGate).toHaveBeenCalledWith(StatsigFeatureGates.SHOW_MULTIPLE_EARN_POOLS)
     expect(getFeatureGate).toHaveBeenCalledWith(StatsigFeatureGates.SHOW_STABLECOIN_EARN)
-    expect(getFeatureGate).toHaveBeenCalledTimes(2)
+    expect(getFeatureGate).toHaveBeenCalledTimes(3)
```

Comment:
this seems unexpected given there are only 2 assertions above, and the multiple pools gate is needed only for the case when the component is used in the token details section. One way you can avoid this is to request the gate more closer to where it is needed.